README.md

```diff
@@ -1,6 +1,68 @@
 # Barefoot Nomad - Frontend
-[![Build Status](https://travis-ci.org/andela/octopus-bn-frontend.svg?branch=develop)](https://travis-ci.org/andela/octopus-bn-frontend)
 
-[![Coverage Status](https://coveralls.io/repos/github/andela/octopus-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/octopus-bn-frontend?branch=develop)
+[![Build Status](https://travis-ci.org/andela/octopus-bn-frontend.svg?branch=develop)](https://travis-ci.org/andela/octopus-bn-frontend) [![Coverage Status](https://coveralls.io/repos/github/andela/octopus-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/octopus-bn-frontend?branch=develop) [![Maintainability](https://api.codeclimate.com/v1/badges/efbdaa199a4e426ce288/maintainability)](https://codeclimate.com/github/andela/octopus-bn-frontend/maintainability)
 
-[![Maintainability](https://api.codeclimate.com/v1/badges/efbdaa199a4e426ce288/maintainability)](https://codeclimate.com/github/andela/octopus-bn-frontend/maintainability)
\ No newline at end of file
+# Description ✍︎
+
+Making company travel and accomodation easy and convenient.
+
+# Dependencies ⚙︎
+
+- React js
+- Node js
+- Redux
+- Webpack
+- SAAS
+- Jest
+- Babel
+- Eslint
+
+# Setting up and running the application 🔧
+
+Clone the repository on your local computer using the command below
+
+> `git clone https://github.com/andela/octopus-bn-frontend.git`
+
+Install all project dependencies using the command below
+
+> `npm i`
+
+To start the development server run the command below
+
+> `npm run dev`
+
+To build the application run the command below
+
+> `npm run build`
+
+To start the application in production mode run the command below
+
+> `npm start`
+
+# Testing 🔬
+
+Install all project dependencies
+
+> `npm i`
+
+Running the tests
+
+> `npm run test`
+
```

Comment:
@AbdoulNuru Would you consider adding an instruction for running coverage?

I mean ```npm run coverage``` to get a test report for the whole codebase

Reply:
done @hakoemmy